HowToREST.md

```diff
@@ -0,0 +1,104 @@
+
+For an example, see the [Cloud Natural Language API](https://cloud.google.com/natural-language/), which is defined by [this Discovery Document](https://language.googleapis.com/$discovery/rest?version=v1). Human-readable documentation is in the [Google Cloud Natural Language API reference](https://cloud.google.com/natural-language/docs/reference/rest/). We’ll call the [AnalyzeEntities](https://cloud.google.com/natural-language/docs/reference/rest/v1/documents/analyzeEntities) service, which takes a block of text as input and returns a list of names and nouns that it finds in the text along with some interesting properties of each entity.
+
+### JSON
+Google REST APIs use JSON for most message responses and POST bodies. Quoting [www.json.org](https://www.json.org/):
+
+> JSON (JavaScript Object Notation) is a lightweight data-interchange format. It is easy for humans to read and write. It is easy for machines to parse and generate. It is based on a subset of the [JavaScript Programming Language](http://crockford.com/javascript/), [Standard ECMA-262 3rd Edition - December 1999](http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf). JSON is a text format that is completely language independent but uses conventions that are familiar to programmers of the C-family of languages, including C, C++, C#, Java, JavaScript, Perl, Python, and many others. These properties make JSON an ideal data-interchange language.
+
+### A way to make API requests
+Google REST APIs are designed to be called by any standards-compliant HTTP client, including the [curl](https://curl.haxx.se/) command-line tool and library.
+
+### Authentication
+To use Google APIs, a client needs to use API keys for unauthenticated requests or OAuth tokens for authenticated requests. For more information, see [the Google Cloud Authentication Overview](https://cloud.google.com/docs/authentication/).
+
+API keys can be obtained from the [Google Cloud Console > Credentials](http://console.cloud.google.com/apis/credentials) page. OAuth tokens can be obtained by [OAuth 2](https://oauth.net/2/) clients and libraries. For a sample command-line client, see the [oauth2l](https://github.com/google/oauth2l) on GitHub.
+
+## An example
+
+Here is a simple example that uses [curl](https://curl.haxx.se/) to call the [AnalyzeEntities](https://cloud.google.com/natural-language/docs/reference/rest/v1/documents/analyzeEntities) service:
+
+```
+$ curl "https://language.googleapis.com/v1/documents:analyzeEntities" \
+  -X POST \
+  -H "X-Goog-Api-Key: $APIKEY" \
+  -H "Content-Type: application/json" \
+  -d '{"document":{"content":"The rain in Spain stays mainly in the plain.", "type":"PLAIN_TEXT"}}' \
+  -i
+HTTP/2 200 
+content-type: application/json; charset=UTF-8
+vary: X-Origin
```

Comment:
I think it is reasonable and likely preferable to exclude everything from lines 41 to 51 in this diff, possibly with an "other headers" placeholder. These headers can and will change.

Reply:
SGTM. Removed.